examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -196,7 +213,9 @@ def main(core_context):
     optimizer = optim.Adadelta(model.parameters(), lr=args.lr)
 
     scheduler = StepLR(optimizer, step_size=1, gamma=args.gamma)
-    for epoch in range(1, args.epochs + 1):
+
+    # NEW: Resume training from epochs_completed. This is useful in the case of pausing and resuming an experiment.
+    for epoch in range(epochs_completed, args.epochs + 1):
 
         steps_completed = epoch * len(train_loader)
```

Comment:
I said this before I think, but it reads awfully strange to set steps_completed before calling `train()`, since at this point you have actually completed zero steps.

Reply:
This is the cleanest way to pass steps_completed into test() though